viewer/src/public/migration/Cognite3DModel.ts

```diff
@@ -224,6 +224,14 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
       throw new Error('NodeId not found');
     }
     const boundingBox3D = response[0].boundingBox;
+    if (boundingBox3D === undefined) {
+      trackError(new Error(`Node ${nodeId} doesn't have a defined bounding box, returning model bounding box`), {
+        moduleName: 'Cognite3DModel',
+        methodName: Cognite3DModel.prototype.getBoundingBoxFromCdf.name
```

Comment:
I would avoid using the name here. The resulting code might be minified/uglified. String feels safer. And shorter :)

Reply:
Good point 👍